backend/dal/Migrations/v01.09.05/Up/PreUp/01-BuildingPredominateUses.sql

```diff
@@ -0,0 +1,50 @@
+PRINT 'Update Building Predominate Uses'
+
+DECLARE @correctPredominateUseId int;
+DECLARE @incorrectPredominateUseId int;
+
+-- Update buildings using wrong predominate use before deleting
+SELECT @correctPredominateUseId = [Id] FROM dbo.[BuildingPredominateUses] where Name = 'Senior Housing (Assisted Living / Skilled Nursing)';
+SELECT @incorrectPredominateUseId = [Id] FROM dbo.[BuildingPredominateUses] where Name = 'Senior Housing (Assisted Living/Skilled Nursing)';
+UPDATE dbo.[Buildings]
+SET [BuildingPredominateUseId] = @correctPredominateUseId
```

Comment:
As it appears you are only using the variable once, there isn't a lot of value in making it.  Might as well just place the query together to make it clearer when reading it for maintainability.
You can do this for those below too.